src/numerics/wrapped_petsc.C

```diff
@@ -26,7 +26,7 @@
 
 // PETSc includes
 #ifdef I
-# define LIBMESH_SAW_I
```

Comment:
I think the indentation here was intentional; we typically do that for preprocessor directives inside preprocessor if/else blocks.

Not that any extra nesting clarity is needed for a one-line block, though.